### ISSUE TITLE
Indent error. (please see example)

### ISSUE THREAD

**Issue by zw963** [zenspider/enhanced-ruby-mode#160]
```rb
new_headers = default_xml_header
          .merge(
          'Authorization' => "Basic #{access_token}", # <= this line indent error?
          'Content-Type' => 'application/vnd.expedia.eps.product-v2+json',
          'Accept' => 'application/vnd.expedia.eps.product-v2+json'
        ).merge(headers.to_h)
```

Thanks

**Comment by zenspider** [comment]
I believe this is fixed with #163 with default variable values. Let me know if you find anything else. Thank you!

**Comment by zw963** [comment]
Yes, it fixed! thank you.
